### ISSUE TITLE
Expose Powergate CID log API

### ISSUE THREAD

**Issue by asutula** [textileio/textile#399]
Currently we're not making this data available through Hub but it will be useful, especially once we start returning better information about job ids and deal states for Bucket archives.

**Comment by jsign** [comment]
Just in case: `hub buck status -w` is basically the old `pow ffs log -w` command

**Comment by asutula** [comment]
You're right, this is likely already supported. I see it in the js client as well. In my upcoming work with updating Powergate version in Hub, I'll double check all the wired are connected here and see if there are any improvements to make.
